core/src/main/java/org/jclouds/proxy/ProxyConfig.java

```diff
@@ -33,9 +33,15 @@
 public interface ProxyConfig {
 
    /**
+    * @deprecated
     * @see org.jclouds.Constants#PROPERTY_PROXY_SYSTEM
-    */
+    */ @Deprecated
    boolean useSystem();
+
+   /**
+    * @see org.jclouds.Constants#PROPERTY_PROXY_FROM_JVM
+    */
+   boolean canUseProxyFromJvm();
```

Comment:
[minor] See comment about naming below: any reason not to call use `useJvm` or `useJvm`?